README.md

```diff
@@ -1,11 +1,11 @@
 # Getting started with Mbed Client on Mbed OS
 
-This is the Mbed Client example for Mbed OS (we also have one for [Linux](https://github.com/ARMmbed/mbed-client-linux-example)). It demonstrates how to register a device with Mbed Device Connector, how to read and write values, and how to deregister. If you are unfamiliar with Mbed Device Connector, we recommend that you read [the introduction to the data model](https://docs.mbed.com/docs/mbed-device-connector-web-interfaces/en/latest/#the-mbed-device-connector-data-model) first.
+This is the Mbed Client example for Mbed OS. It demonstrates how to register a device with Mbed Device Connector, how to read and write values, and how to deregister. If you are unfamiliar with Mbed Device Connector, we recommend that you read [the introduction to the data model](https://docs.mbed.com/docs/mbed-device-connector-web-interfaces/en/latest/#the-mbed-device-connector-data-model) first.
```

Comment:
This content about Mbed Device Connector is old. Could we please remove it (if not in this PR than in another one - I don't want to prevent this PR from merging)?

Reply:
Do you think we could update the link and point to this one?
https://cloud.mbed.com/docs/latest/legacy-products/mbed-device-connector-web-api.html

Reply:
I think that's better, as long as they're public. Are there plans to update this example in the future so it doesn't use Device Connector, which we list as a legacy product?

Reply:
I don't believe we'll update this example - this works specifically with Device Connector, and we need to support it (at least for a few more releases). I'll update the link and suggest we merge it.
We should probably talk offline to understand the concerns you have on this doc (I've realized is the same content but different link).

Reply:
Sounds like a plan